poly/src/domain/radix2/fft.rs

```diff
@@ -137,21 +137,53 @@ impl<F: FftField> Radix2EvaluationDomain<F> {
             });
     }
 
+    #[allow(unused)]
```

Comment:
This is not unused, right?

Reply:
root_len is unused in the parallel cae

Reply:
and roots needs `allow(unused_mut)` in the parallel case as well

Reply:
Maybe let's wrap it in a `#[cfg(not(feature = "parallel"))]`?